src/organization/OrganizationDTO.ts

```diff
@@ -140,5 +140,7 @@ export class OrganizationDTO implements Organization {
   @IsOptional()
   parentOrg?: Organization;
 
+  subOrgs?: Organization[];
```

Comment:
Should this have the `@IsOptional()` tag as well?

Reply:
👍